server/src/config.ts

```diff
@@ -65,6 +65,7 @@ export default {
   emailTransportTypes: process.env.EMAIL_TRANSPORT_TYPES || null as string | null,
   encryptionPassword: process.env.ENCRYPTION_PASSWORD_00001 as string,
   fbAppId: process.env.FB_APP_ID || null as string | null,
+  logTransport: process.env.SERVER_LOG_TRANSPORT as string,
```

Comment:
This may have snuck in due to the `squash-merge` process mucking up the diffs when I merged `edge` into my branch :-/

Reply:
yeah you can remove that line